README.client.rst

```diff
@@ -1,9 +1,35 @@
-Rucio Client Lite Package
+Rucio: Client Lite Package
 =========================
 
+Rucio is a project that provides services and associated libraries for allowing scientific
+collaborations to manage large volumes of data spread across facilities at
+multiple institutions and organisations. Rucio has been developed by
+`the ATLAS experiment <https://atlas.cern/>`_. It offers advanced features, is
+highly scalable and modular. Rucio is a data management
+solution that could cover the needs of different communities in the scientific
+domain (e.g., HEP, astronomy, biology).
 
-Development takes place via the usual Rucio processes as outlined in `<http://rucio.cern.ch/>`_.
 
-The master repository is on `<https://gitlab.cern.ch/rucio01/rucio>`_.
+Documentation
+-------------
 
-See release notes and more at `<https://its.cern.ch/jira/browse/RUCIO>`_.
+General information and latest documentation about Rucio can be found
+at http://rucio.readthedocs.io.
+
+Developers
+----------
+
+For information on how to contribute to Rucio, please refer and follow our
+`Guidelines <http://rucio.readthedocs.io/contributing.html>`_ or CONTRIBUTING_.
```

Comment:
I would only link to CONTRIBUTING_ here. In the CONTRIBUTING then we can have more links.